spec/views/case_workers/claims/show_spec.rb

```diff
@@ -322,6 +333,8 @@
     end
 
     context 'when the claim was created before messaging feature released' do
+      # The majority of this test has been moved to the case_workers/claims/_claim_status.html.haml
+      # as the show page has now been split across two tabs claim_status and information_status
```

Comment:
I'd also possibly consider removing this comment - it was helpful to understand the change when reviewing the PR but I'm not sure it needs to be in the code long-term. Maybe it would have been better in the commit message?